stock_inventory_revaluation/models/stock_inventory_revaluation_line.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockInventoryRevaluationLine(models.Model):
+    _name = "stock.inventory.revaluation.line"
+    _description = "Stock inventory revaluation Line"
+
+    name = fields.Char("Description")
+    stock_inventory_revaluation_id = fields.Many2one(
+        "stock.inventory.revaluation",
+        "Inventory revaluation",
+        required=True,
+        ondelete="cascade",
+    )
+
+    product_id = fields.Many2one("product.product", "Product", required=True)
+    account_id = fields.Many2one(
+        "account.account", "Account", domain=[("deprecated", "=", False)]
+    )
+    currency_id = fields.Many2one(
+        "res.currency", related="stock_inventory_revaluation_id.currency_id"
+    )
+    stock_move_id = fields.Many2one("stock.move", "Stock Move", readonly=True)
+    quantity = fields.Float("Quantity", default=1.0, digits=0, required=True)
+    original_value = fields.Monetary("Original Value")
+    price_subtotal = fields.Monetary("Total Cost", required=True)
```

Comment:
I not sure if is required for any purpose, but is field is zero i get error
![image](https://user-images.githubusercontent.com/7775116/156857702-b775660e-51a5-4fbd-ae8d-76c2c778cc77.png)

Reply:
It is needed for the calculation, so I decided to add a default value of 0.0